src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -578,22 +649,29 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
     }
   }
 
-  function scheduleWork(root : FiberRoot) {
-    if (defaultPriority === SynchronousPriority) {
-      throw new Error('Not implemented yet');
+  function scheduleWork(root : FiberRoot, priorityLevel : ?PriorityLevel) {
```

Comment:
We should ideally not use `undefined` since this doesn't let us track accidental missing argument. `null | PriorityLevel` would be better.

Ideally we shouldn't have an nullable numbers. This causes them to be boxed (in more optimized compilers than JS VMs).

Maybe there's a way to restructure this into two methods where the other calls this with `priorityContent`?